components/docs-chef-io/content/habitat/studio.md

```diff
@@ -33,73 +22,90 @@ The desire to build Habitat packages in CI leads to the question of why we requi
+| Linux Native | Yes | No | No |
+| Linux Docker | Yes | Yes | Yes |
+| Windows Native | No | No | Yes |
+| Windows Docker | No | No | Yes |
 
 ### Why do we need privileged containers to build (on Linux)?
+
 Users wishing to build Habitat packages in containers will typically have their CI agent invoke `hab pkg build*`. This agent will be running in a non-privileged container, or possibly create a non-privileged container to invoke `hab pkg build`.
 
-At this point, the hab command will try to create the studio to run the command in. In the case of the native studio, this will fail because the container it is running in does not have the `CAP_SYS_CHROOT` or `CAP_SYS_ADMIN` capabilities.  The Docker studio will also fail, as the environment the build is executing in won't have permissions to run containers,  and also wouldn't have access to the docker socket (or docker in docker).
+At this point, the hab command will try to create the studio to run the command in. In the case of the native studio, this will fail because the container it is running in does not have the `CAP_SYS_CHROOT` or `CAP_SYS_ADMIN` capabilities.  The Docker studio will also fail, as the environment the build is executing in won't have permissions to run containers,  and also wouldn't have access to the Docker socket (or Docker in Docker).
 
-It's possible that if the users are able to provide credentials to access a remote docker host, we would be able to build, though in secured environments this is probably unlikely. I also don't know if volume mapping can work across the network. Even if it is possible, any IO operations, such as reading the source would be slow.
+It's possible that if the users are able to provide credentials to access a remote Docker host, we would be able to build, though in secured environments this is probably unlikely. I also don't know if volume mapping can work across the network. Even if it is possible, any IO operations, such as reading the source would be slow.
 
 We also want to be careful about exposing build functionality directly to CI agents without requiring the studio boundary.  We'd likely start (for example) fielding support questions around our package quality from user software segfaulting, when the underlying issue was the package was built on an Ubuntu host and it linked against the wrong libraries.
 
-Today, users can configure their agents to run the docker studio image directly, but that requires them to perform all the set up `hab pkg build` would normally do for them, creating a point of friction.
+Today, users can configure their agents to run the Docker studio image directly, but that requires them to perform all the set up `hab pkg build` would normally do for them, creating a point of friction.
 
-* `hab studio build` is a synonym for `hab pkg build`
-* Does windows suffer from this same issue?
+- `hab studio build` is a synonym for `hab pkg build`
+- Does Windows suffer from this same issue?
```

Comment:
I think you could remove these two lines. They look like they were internal notes.